packages/cloud-cognitive/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -368,42 +368,43 @@
   flex-direction: row;
 }
 
-.#{$block-class}__datagridLeftPanel {
-  display: flex;
-  width: 388px; // as per UX specs.
-  background-color: $layer-01;
-}
-
 .#{$block-class}__leftPanel-position {
   display: inherit;
 }
 
 .#{$block-class}__datagridWithPanel {
   position: relative;
   display: flex;
-  width: 100%;
   height: 100%;
   flex-direction: column;
 
   .#{$block-class}__grid-container {
     display: flex;
-    overflow: hidden;
+    overflow: visible;
     flex-direction: column;
   }
 
   .#{$block-class}__table-toolbar {
     width: 100%;
   }
 
-  .#{$block-class}__datagridLeftPanel {
-    flex: 0 0 auto;
+  .#{$block-class}__table-container {
+    display: flex;
+    overflow: visible;
+    background-color: $layer-01;
   }
 
-  .#{c4p-settings.$carbon-prefix}--data-table-content {
-    flex: 1 1 0%;
-  }
-  .#{$block-class}__table-container {
+  .#{$block-class}__table-container-inner {
     overflow: hidden;
+    flex: 1;
+  }
```

Comment:
It looks like this is what’s causing some of the tables not to take up the full width.

![image](https://user-images.githubusercontent.com/1717398/219826395-9475c137-47a3-4065-898c-edb66c8a7b5f.png)

Since this is nested under `.#{$block-class}__datagridWithPanel`, on stories without the panel, this no longer fills the space. We can either add a conditional to keep this inner container to cases with the filter panel, or move this outside of the `.--datagridWithPanel` block.